.contributing/the-process.md

```diff
@@ -96,13 +96,23 @@ Create: 
 
         -   Decide on widget/action/middleware names
 
--   An issue on GitHub for each use case
+-   An issue on GitHub for each use case:
 
-    1.  Add a section label (eg. Auth) 
+    1.  Create a tracking issue
 
-    2.  Indicate the use case it addresses & link to the Google doc  
+        - add the "tracking" label
 
--   Links back and forth between the Task, Issue & Doc 
+        - Add a title "section_name use_case_number: use_case_name"
+
+        - in the description, break down the use case into a checklist of tasks (sub issues)
+
+        - grab team for discussion/ input on checklist
+
+    2.  For each task in tracking issue, create an issue and link its issue number
+
+    3.  Add all created issues to appropriate github projects
+
+- Add Links back and forth between the Task, Issue & Doc
```

Comment:
Uber ultra nitttiest of nits: could this be "Add links"?  with lowercase l?

Reply:
![](https://www.guinnessworldrecords.com/Images/Betsy%20Bond%20-%20Largest%20Knitting%20Needles%20-%20header_tcm25-541058.jpg)

Reply:
lol your alg :) im on it!

Reply:
Hahaha super nit!  Love it!

Reply:
Does the team need to weigh in on the sub issue checklist? We can always edit and comment later if need be

Reply:
@T-Davis is that a question for Tam? Or both of us?  If me, could you clarify?  I don't follow sorry

Reply:
My bad, it looked different in vscode, moved the comment to the appropriate line

Reply:
Got it, thanks! 👍